src/components/TodoApp.vue

```diff
@@ -1,35 +1,40 @@
 <template>
-  <ToDoForm @addToDo="addToDo" @clearToDo="clearToDo"/>
+  <ToDoHeader @showForm="toggleForm" />
+  <ToDoForm v-show="isShowingForm" @addToDo="addToDo" @clearToDo="clearToDo" @closeForm="toggleForm" />
+  <EmptyListImage v-if="isShowingEmptyImage" class="ml-auto mr-auto"/>
   <ToDoList :todos="todos" @deleteToDo="removeToDo" />
 </template>
 
 <script setup lang="ts">
-  import ToDoForm from './ToDoForm.vue'
-  import ToDoList from './ToDoList.vue'
-  import { ref, Ref } from 'vue';
-  const todos: Ref<string[]> = ref([]) 
-
-  function addToDo(todo: string) {
-    if (todos.value.includes(todo)) {
-      return
-    }
-
-    todos.value.push(todo)      
-  }
-
-  function removeToDo(item: string) {
-    const index = todos.value.indexOf(item)
-    if (index > -1) {
-      todos.value.splice(index, 1)
-    }
+import ToDoForm from './ToDoForm.vue'
+import ToDoList from './ToDoList.vue'
+import ToDoHeader from './ToDoHeader.vue'
+import EmptyListImage from './EmptyListImage.vue'
+import { Todo } from '../todo.ts'
+import { ref, Ref, computed } from 'vue'
+const todos = ref<Todo[]>([])
+const isShowingForm: Ref<boolean> = ref(false)
+const isShowingEmptyImage = computed(() => !isShowingForm.value && !todos.value.length)
+
+function addToDo(todo: Todo) {
+  todos.value.push(todo)
+  isShowingForm.value = !isShowingForm.value
```

Comment:
You have a function for this, call that function instead of writing the same thing